src/styles/todo.scss

```diff
@@ -100,3 +100,52 @@
     opacity: 0.5;
   }
 }
+
+.item {
+  &-enter {
+    max-height: 0;
+
+    &-active {
+      overflow: hidden;
+      max-height: 58px;
+      transition: max-height 0.3s ease-in-out;
+    }
```

Comment:
I think, something is wrong with your BEM.
Reed/watch more [here](https://en.bem.info/methodology/naming-convention/) and [here](https://mate.academy/learn/css-advanced/bem#/video/920)
If you want, come to qna and we would discuss it there to clear this topic.